src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -41,6 +41,56 @@ namespace Utils
             m_db.reset(dbRawPtr);
         }
 
+        /**
+         * @brief Copy constructor.
+         *
+         * @param other Other instance.
+         */
+        RocksDBWrapper(RocksDBWrapper& other)
```

Comment:
> This way, we remove the possibility of creating a real copy of this class.

I think that we can't really have copies of this class, because the member variable is an `unique_ptr` and not a `shared_ptr`.

> Couldn't be defined copy and move constructors and then be invoked in the program using the one that is expected to?

Not sure if I understand your suggestion, sorry. Do you say we simply delete this copy constructor? How would be a correct copy constructor for this class that doesn't involve a move?
It seems that I can make a deep copy or directly change it to `shared_ptr`, but it might not be the original idea of the author of the class.
https://stackoverflow.com/questions/16030081/copy-constructor-for-a-class-with-unique-ptr

Reply:
For now, I'm deleting both copy constructor and copy assignment.

Reply:
You are right that the copy constructor is not allowed since it has unique_ptr. I agree with the solution implemented.

Reply:
Thank you!